src/js/other-websites/add_input_icon.js

```diff
@@ -24,9 +24,29 @@ function addRelayMenuToPage(relayMenuWrapper, relayInPageMenu, relayIconBtn) {
 function positionRelayMenu() {
   const relayInPageMenu = document.querySelector(".fx-relay-menu-iframe");
   const relayIconBtn = document.querySelector(".fx-relay-menu-open");
+
+  if (!relayIconBtn) {
+    // BUGFIX: Catch erroneous/duplicate positionRelayMenu() calls before menu is open
+    return;
+  }
```

Comment:
Note: I was getting duplicate/race-condition like calls to the `positionRelayMenu()` function, so I needed a way to drop them if the menu wasn't open. This solves the symptom, not the issue.